catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -370,6 +373,12 @@ private void deleteRegistryConfigurations(Metacard metacard)
                 RegistryObjectMetacardType.REGISTRY_ID,
                 registryId));
         if (configurations != null) {
```

Comment:
According to the interface javadoc, ConfigurationAdmin.listConfigurations will only return a non-empty list or null. Should I still add a size check?